app/cdap/components/Administration/TetheringTabContent/reducer.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Dispatch } from 'react';
+import { IAction } from 'services/redux-helpers';
```

Comment:
@njbriggs I'm sure theres an @types/redux or something like that right? - we should use that instead of creating our own interfaces or types for libraries

Reply:
created [this](https://cdap.atlassian.net/browse/CDAP-18770) JIRA ticket to track it.